### PR TITLE
[TASK] Add favicon generator (incl. app tiles) and logo

### PR DESCRIPTION
**Pull request by xf-**
## Description

Add logo and tiles. Also add generator API for your own logo.

## Motivation and Context

Nice splash screen or tiles
## How Has This Been Tested?

Needs gulp

## Screenshots/links (if appropriate):
screenshot_20170324-235604
screenshot_20170324-235542

## Checklist:

- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)
- [X] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: gulp/tasks/favicon.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+module.exports = function (gulp, plugins, config) {
+  return function javascript(cb) {
+    plugins.realFavicon.generateFavicon({
+      masterPicture: 'assets/logo.svg',
+      dest: 'assets/favicon',
```
Comment (genofire) [inline gulp/tasks/favicon.js:5]: Hi should `assets/favicon` not be in the `.gitignore` and the files with these path not in this repo?

Comment (xf-) [reply gulp/tasks/favicon.js:5]: i want the icons in the repo.
The API can generate the icons, but needs to be up and is a 3rd-party service.

Comment (genofire) [reply gulp/tasks/favicon.js:5]: Is there a way of offline generation?

Comment (xf-) [reply gulp/tasks/favicon.js:5]: you can build one ;) i'm happy with api. much easier to update.

Comment (xf-) [reply gulp/tasks/favicon.js:5]: OK so?

Comment (genofire) [reply gulp/tasks/favicon.js:5]: unglücklich aber ja

**Review by genofire — APPROVED** [review]
externe build api needed :-1:

**Comment by xf-** [comment]
@genofire you can build it yourself and replace the icons. Only the inject markup is used in gulp process. Generate favicon is separate task and will use the logo.

**Pull request merged**